src/testlib/include/testlib/yamlTest.hpp

```diff
@@ -18,8 +18,13 @@
 #include <string>
 #include <vector>
 
+#include <boost/exception/diagnostic_information.hpp>
+
 #include <yaml-cpp/yaml.h>
 
+#include <testlib/findRepoRoot.hpp>
```

Comment:
Are these two headers needed?

Reply:
Ya the functions they export are used further down - didn't catch it before because the valgen yaml tests were including them on their own (but hey now I can remove those includes from there so did that as well)